### ISSUE TITLE
Make the documentation more accessible

### ISSUE THREAD

**Issue by adamtheturtle** [ClusterHQ/flocker#418]
The current Sphinx theme (as of reviewing https://github.com/ClusterHQ/flocker/pull/413) has various accessibility issues. We should define our minimum standards for accessibility and fix any violations of these standards.

See https://github.com/ClusterHQ/flocker/pull/413#issuecomment-51083608
